app/src/main/res/drawable/drop_shadow.xml

```diff
@@ -19,21 +21,3 @@
         </shape>
     </item>
 </layer-list>
-
-<!--<?xml version="1.0" encoding="utf-8"?>-->

```

Comment:
So this was just extra in the end? Or did you mean to move it somewhere else?

Reply:
It was an experiment that I forgot to delete

Reply:
Ok, looks fine to me